assets/scss/modules/qna.scss

```diff
@@ -243,6 +255,16 @@ table.qna-list-table {
 				font-weight: bold;
 				font-size: 16px;
 				color: var(--tutor-body-color);
+
+				ul {
+					padding-left: 2rem;
+
+					li {
+						list-style-type: disc;
```

Comment:
@mi-prakash vai, any reason to specify ```font-size``` for list item? If we have text on a TD and a list item on another TD, it may look odd. If we don't specify then it will take font size according to the theme.

Reply:
ok removing the font size.
I used it because on the chat page font was used as 13 px on the list items